Doc/c-api/init.rst

```diff
@@ -1048,6 +1048,11 @@ All of the following functions must be called after :c:func:`Py_Initialize`.
    Reset all information in a thread state object.  The global interpreter lock
    must be held.
 
+   .. versionchanged:: 3.9
+      This function is now responsible to call
+      :c:member:`PyThreadState.on_delete` callback. Previously,
+      :c:func:`PyThreadState_Delete` was responsible for that.
```

Comment:
This usage of "responsible" could be confusing.  Consider instead:

```
   This function now calls the :c:member:`PyThreadState.on_delete`
   callback.  Previously, that happened in :c:func:`PyThreadState_Delete`.
```

Reply:
Thanks, I updated the doc.